app/src/main/kotlin/com/apkupdater/data/github/GitHubApp.kt

```diff
@@ -83,5 +83,6 @@ val GitHubApps = listOf(
     GitHubApp("com.shadow.blackhole", "Sangwan5688", "BlackHole"),
     GitHubApp("com.zionhuang.music", "z-huang", "InnerTune"),
     GitHubApp("com.nextcloud.client", "nextcloud", "android"),
-    GitHubApp("at.bitfire.davdroid", "bitfireAT", "davx5-ose")
+    GitHubApp("at.bitfire.davdroid", "bitfireAT", "davx5-ose"),
+    GitHubApp("keepass2android.keepass2android", "PhilippC", "keepass2android")
```

Comment:
As it is, this won't work properly. The releases contain 2 apks with different package names and there's no way to know which one is the correct.

Reply:
I assume you're talking about the 2 different APKs that are included as asset [here](https://github.com/PhilippC/keepass2android/releases/tag/v1.09e-r7)? One is the online version and the nonet version is the offline version.

It's simple: we can add an extra entry as they have different package names or am I missing something here? See [here](https://www.apkmirror.com/apk/philipp-crocoll-croco-apps/) as well.

Reply:
That won't work because it will always retrieve the first apk. Some extra functionality needs to be added, like a mandatory string match on the filename.

Reply:
And on top of this, this app uses absurd version names that are not going to match properly.

Reply:
Hmm, so we can do two things regarding this PR:

1. Wait until APKUpdater will get a feature to match the string, but that's up to you
2. Close this PR because the version names don't make much sense

It's up to you.